src/components/gw-user-options.vue

```diff
@@ -124,6 +124,8 @@ export default {
 
         img {
             width: 100%;
+            min-width: 100%;
+            min-height: 100%;
```

Comment:
These are not necessary.
The width of the image is already defined as 100%.
The reason for not having a height is to allow correct image proportions. Though I can see the issue with that here.

Also, in CSS there is no reason to use `min-width` or `min-height` as 100%. Just use `width` and `height`.

I think what we need to do is add `overflow: hidden;` to the `.profile-image` parent class.

Reply:
@jesusantguerrero Please check the comment if you haven't.

Reply:
> @jesusantguerrero Please check the comment if you haven't.

@SparoHawk  done